tests/integration/nvidia_tuned/test_prepare_nvidia_profiles.py

```
@@ -834,3 +834,108 @@ def test_prepare_nvidia_profiles_common_profiles_deployed(base_image):
+    _require_ubuntu_2404(base_image)
+    runner = DockerTestRunner(package="nvidia-tuned", base_image=base_image)
+    try:
+        configmaps = {"accelerator": "vr200", "intent": intent}
+        try:
+            runner.run_script(
+                script="prepare_nvidia_profiles.sh",
+                configmaps=configmaps,
+                skip_system_operations=True,
+            )
+        except Exception:
+            pass
```

Ruff (0.15.15)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)